src/components/AssetsFilters/index.jsx

```diff
@@ -30,18 +31,23 @@ const ASSET_TYPES = [
 ];
 
 export const AssetsFilters = ({ assetsFilters, updateFilter }) => (
-  <ul className={styles['filter-set']}>
-    {ASSET_TYPES.map(type => (
-      <li key={type.key}>
-        <CheckBox
-          name={type.key}
-          label={type.displayName}
-          checked={assetsFilters[type.key]}
-          onChange={checked => updateFilter(type.key, checked)}
-        />
-      </li>
-    ))}
-  </ul>
+  <div role="group" aria-labelledby="filter-label">
+    <h4 id="filter-label" className={styles['filter-heading']}>Filter by File Type</h4>
+    <div className={styles['filter-set']}>
+      <CheckBoxGroup>
+        {ASSET_TYPES.map(type => (
+          <CheckBox
+            key={type.key}
+            id={type.key}
+            name={type.key}
```

Comment:
I didn't follow the discussion around these changes and the CheckBoxGroup while they were happening, so please correct me if I'm wrong. When we were at the Accessibility Unmeeting before the winter holidays, Mark said that the checkbox's ids should all be unique (and they were all the same at the time), and that the names should all be the same (my notes say something like "filters"). Is that correct? Does the CheckBoxGroup solve that problem? I'm wondering whether the name attribute still being unique to the checkbox is a problem?

Reply:
Changing this so that all of the names are the same does not affect functionality - let me sanity check on having those be the same is necessary for Accessibility and get back to you with the final answer here with regards to the `name` attribute :) I do remember that the `id` needed to be unique for sure.

Reply:
The examples here https://www.w3.org/WAI/tutorials/forms/grouping/ show that the name attribute is the same for radio buttons, but not for checkbox inputs. I think we're okay to keep this unique, but will check in with @cptvitamin for final verification. 

<img width="852" alt="screen shot 2018-01-09 at 11 09 29 am" src="https://user-images.githubusercontent.com/9357438/34730509-9cbcd4b6-f52d-11e7-829b-954f2a17e18b.png">

Reply:
Confirmed -- leaving this as is for now. Looking at the latest spec https://www.w3.org/TR/html52/sec-forms.html#checkbox-state-typecheckbox - the name attribute is needed for Radio buttons, but isn't required to be identical for checkboxes.

Reply:
That sounds good. Thanks a lot for following up on that!